chore(changelog): update changelog for v3.2.0 (#407)
newrelic/nri-kubernetes: CHANGELOG.md

The format is based on [Keep a Changelog](http://keepachangelog.com/)
and this project adheres to [Semantic Versioning](http://semver.org/).

## 3.2.0

### Added

> * sink: implement mTLS-enabled http client by @roobre in https://github.com/newrelic/nri-kubernetes/pull/338

Reviewer: ```suggestion
* Initial, internal-only implementation of mTLS-enabled sink by @roobre in https://github.com/newrelic/nri-kubernetes/pull/338
```

> * Feature/restart count delta by @sigilioso in https://github.com/newrelic/nri-kubernetes/pull/382
>   a New metric has been added. It reports the number of restart since last sample: `restartCountDelta`

Reviewer: ```suggestion
* `restartCount` metric for pods is now also available as `restartCountDelta` by @sigilioso in https://github.com/newrelic/nri-kubernetes/pull/382
```

> * Dockerfile: simplify binary path, ensure 755 permissions by @roobre in https://github.com/newrelic/nri-kubernetes/pull/364

Reviewer: ```suggestion
```

Reviewer: Probably not worth mentioning on the Change log user-facing impact is nil

### Fixed

> * fix(pendingPods): some metrics are missing from KSM by @paologallinaharbur in https://github.com/newrelic/nri-kubernetes/pull/404
>   The metric `isReady` is now reported for pending pod as well

Reviewer: ```suggestion
* `isReady` metric is now correctly reported as `false` (rather than `NULL`) for pending pods by @paologallinaharbur in https://github.com/newrelic/nri-kubernetes/pull/404
```
